src/components/organisms/cards/cards.html

```diff
@@ -0,0 +1,43 @@
+<div class="container">
+  <div class="card">
+    <figure class="wrap">
```

Comment:
```suggestion
    <figure class="overwrap">
```
3枚のカードいずれも`overwrap`ではなく`wrap`になってしまっているため、写真部と文章部が大幅にずれていそうです。修正お願いします
![image](https://github.com/JuN-front/Junfuku-frontend-basic-last-assignment/assets/104500220/74eb45e8-d4c1-4bed-8975-89a99e91ff7b)